Brokerages/DefaultOrderBook.cs

```diff
@@ -17,19 +17,19 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace QuantConnect.Brokerages.GDAX
+namespace QuantConnect.Brokerages
 {
     /// <summary>
     /// Represents a full order book for a security.
     /// It contains prices and order sizes for each bid and ask level.
     /// The best bid and ask prices are also kept up to date.
     /// </summary>
-    public class OrderBook
+    public class DefaultOrderBook : IOrderBook<decimal, decimal>
     {
         private readonly object _locker = new object();
         private readonly Symbol _symbol;
-        private readonly SortedDictionary<decimal, decimal> _bids = new SortedDictionary<decimal, decimal>();
-        private readonly SortedDictionary<decimal, decimal> _asks = new SortedDictionary<decimal, decimal>();
+        protected readonly SortedDictionary<decimal, decimal> _bids = new SortedDictionary<decimal, decimal>();
+        protected readonly SortedDictionary<decimal, decimal> _asks = new SortedDictionary<decimal, decimal>();
```

Comment:
To follow the coding style used in our repo, only private class members should start with an underscore.
Please rename `_bids` to `Bids` and `_asks` to `Asks`